cunumeric/array.py

```diff
@@ -506,7 +578,7 @@ def flags(self) -> Any:
         for C-style contiguous arrays or ``self.strides[0] == self.itemsize``
         for Fortran-style contiguous arrays is true.
         """
```

Comment:
Let's add a note here (ideally near the top of the docstring) that, with the exception of "writeable", these flags don't reflect the properties of the cuNumeric array, but rather the NumPy array that will be produced if the cuNumeric array is pulled to a single node. And probably also note this on `setflags`.

Reply:
https://github.com/nv-legate/cunumeric/pull/759/commits/05c05655591aecc6bbc89d81c6a0ed8a98400021